src/melee/ft/chara/ftMario/ftMario_SpecialLw.c

```diff
@@ -274,8 +275,9 @@ void _ftMario_800E25C4_800E2778_helper(HSD_GObj* gobj)
     }
 }
 
-static ftCollisionBox ftMario_SpecialLw_CollisionBox = { 12.0F, 0.0F, -6.0F,
-                                                         6.0F,  6.0F, 6.0F };
+static ftCollisionBox ftMario_SpecialLw_CollisionBox = {
+    12.0F, 0.0F, { -6.0F, 6.0F }, { 6.0F, 6.0F }
+};
```

Comment:
`clang` was emitting a warning for not grouping the sub-structures. Can be fixed automatically via `clang-tidy`/LSP so I think it's fine to comply.